src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="ReadOnlySpan\BinarySearch.cs" />
```

Comment:
nit: can you please move this into the ReadOnlySpan test property group?

Also consider splitting the tests into separate files for Span and ReadOnlySpan tests, but up to you.